pom.xml

```diff
@@ -9,10 +9,15 @@
         <version>2.17</version>
     </parent>
 
-    <name>Jenkins BlueOcean Pipeline Editor</name>
+    <name>Blue Ocean Pipeline Editor</name>
+    <description>
+        The Blue Ocean Pipeline Editor is the simplest way for anyone 
```

Comment:
As you've wrapped these on different lines I do not know how the plugin manager will format this description.

I suspect it might `way for anyonewanting to`